schema/bertron/linkml/search_result.yaml

```diff
@@ -0,0 +1,280 @@
+# yaml-language-server: $schema=https://linkml.io/linkml-model/linkml_model/jsonschema/meta.schema.json
+id: https://github.com/ber-data/bertron-schema/schema/bertron/linkml
+name: ber_data
+description: Schema for BERtron common data model.
+version: 0.0.1
+prefixes:
+  bioportal: https://bioportal.bioontology.org/ontologies/
+  biolink: https://w3id.org/biolink/vocab/
+  kbcms: https://kbase.github.io/credit_engine/
+  crcr: https://credit.niso.org/contributor-roles/
+  Crossref: https://crossref.org/
+  DataCite: https://purl.org/datacite/v4.4/
+  DOI: http://identifiers.org/doi/
+  gdmt: http://vocab.fairdatacollective.org/fdc-gdmt/en/page/
+  JGI: https://data.jgi.doe.gov/search/
+  linkml: https://w3id.org/linkml/
+  ORCID: http://identifiers.org/orcid/
+  OSTI.ARTICLE: https://www.osti.gov/biblio/
+  ROR: http://identifiers.org/ror/
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  schema: http://schema.org/
+  xsd: http://www.w3.org/2001/XMLSchema#
+imports:
+  - linkml:types
+default_range: string
+default_curi_maps:
+  - semweb_context
+default_prefix: bertron
+
+classes:
+
+  Entity:
+    tree_root: true
+    description: |
+      An object retrieved by BERtron from a BER data API.
+    attributes:
+      ber_data_source:
+        description: The BER member from whence the entity originated.
+        range: BERSourceType
+        required: true
+      coordinates:
+        description: The geographic coordinates associated with an entity. For entities with a bounding box, the centroid is used as the geographic references.
+        range: Coordinates
+        required: true
+      curie:
+        description: Compact URI for the entity.
+        range: curie
+        required: false
+      data_source_internal_identifier:
+        description: The unique ID used for the entity within the BER entity. It may not necessarily be resolvable.
+        identifier: true
+        range: string
+        required: true
+      data_type:
+        description: What kind of data this entity is -- e.g. sequence data; a soil core; a well; etc.
+        range: DataTypeTag
+        multivalued: true
+      description:
+        description: Textual description of the entity.
+        required: false
+      identifiers:
+        description: Fully-qualifier URL or CURIE used as an identifier for an entity.
+        range: uriorcurie
+        required: true
+        slot_uri: schema:identifier
+        examples:
+          - value: "NCBItaxon:172684329"
+          - value: "ISGN:1986497"
+      names:
+        description: Textual identifiers for an entity.
+        multivalued: true
+        range: Name
```

Comment:
may be sufficient just to have a list of strings here

alternatively, `Name` could be a class with a name value (the string) and name type, e.g. primary name, synonym (exact/broad/narrow/related), etc.